src/components/TransactionsList.js

```diff
@@ -1,6 +1,24 @@
 import React from 'react'
+import Transaction from './Transaction'
 
-const TransactionsList = () => {
+const TransactionsList = (props) => {
+
+  var transactionItems = []
```

Comment:
I probably prefer to do this filtering on the container - that way you can split this into separate methods. 

Also, it's probably easier to do this using `filter` and `map` - I would recommend trying to refactor to use those methods